.luacheckrc

```diff
@@ -138,96 +138,29 @@ add_ignore("CorsixTH/Lua/rooms/operating_theatre.lua", "wait_for_ready")
 
 -- W111: setting non-standard global variable XYZ
 -- W113: accessing undefined variable XYZ
--- W121: setting read-only global variable XYZ
 -- W122: mutating read-only global variable XYZ
--- W211: unused variable XYZ
 -- W212: unused argument XYZ
--- W231: variable XYZ is never accessed
--- W542: empty if branch
```

Comment:
I'd keep these comment lines incase they end up being needed in the future

Reply:
That's what luacheck's documentation is for - https://luacheck.readthedocs.io/en/stable/warnings.html

I agree with removal. Maybe link the document.